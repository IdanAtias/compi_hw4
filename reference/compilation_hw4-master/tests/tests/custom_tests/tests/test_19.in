matrix m1(3,3) = [1, 2, 3; 4, 5, 6; 7, 8, 9];
matrix m2(2,3) = [1, 2, 3; 4, 5, 6];
// Should work
m1 :: transpose(m2);
// Testing wrong dimensions
m1 :: m2;

