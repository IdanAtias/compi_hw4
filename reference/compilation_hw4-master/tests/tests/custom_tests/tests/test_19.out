m1=
1,2,3
4,5,6
7,8,9
m2=
1,2,3
4,5,6
1,2,3,1,4
4,5,6,2,5
7,8,9,3,6
ArgumentSizeMismatch with operation ::. left argument size 3 and right argument size 2
